shakedown/dcos/__init__.py

```diff
@@ -129,6 +131,20 @@ def authenticate_oauth(oauth_token):
         return None
 
 
+def attach_existing_cluster(url):
+    """Attach an existing (already setup) cluster.
+    return: cluster object, or None
+    """
+    clusters = [c.dict() for c in dcos.cluster.get_clusters()]
+    for c in clusters:
+        if url == c['url']:
+            try:
+                cluster.set_attached(dcos.cluster.get_cluster(c['name']).get_cluster_path())
+                return cluster
+            except:
+                return None
+
```

Comment:
Should have a default `return None` after the `for` loop incase `url` does not match any known cluster.